darwin/datatypes.py

```diff
@@ -104,3 +141,11 @@ def make_video(keyframes, start, end):
             "segments": [[start, end]],
         },
     )
+
+
+def make_video_annotation(frames, keyframes, segments, interpolated):
+    first_annotation = list(frames.values())[0]
+    if not all(frame.annotation_class.name == first_annotation.annotation_class.name for frame in frames.values()):
+        raise ValueError("invalid argument to make_video_annotation")
```

Comment:
Makes sense. Two questions:

1. Has this inconsistency ever happened to our data? That'd be pretty bad.
2. Do we gain anything by having yet another custom exception for these cases? If so, it's worth defining one.

Reply:
It's more to protect users of the SDK to build invalid annotations